grammars/basic.cson

```diff
@@ -26,7 +26,7 @@ patterns: [
   }
   {
     # comment
-    match: '(?i:REM.*)'
+    match: '(?i:\bREM\b.*|^'.*| '.*)'
```

Comment:
👋 from [github/linguist](https://github.com/github/linguist) which uses this grammar for highlighting BASIC files on GitHub.com.

This PR introduces a typo which causes our grammar compiler to fail because the new `'` chars here aren't escaped which makes this an invalid CSON grammar file.

This should probably be `match: '(?i:\bREM\b.*|^\'.*| \'.*)'

Reply:
Thank you, @lildude. I've made a new PR (#2) to address this feedback.

Hoping to see the new comment syntax support make it to the GitHub syntax highlighter!